packages/bot-web-ui/src/components/dashboard/tutorial-tab/index.scss

```diff
@@ -156,3 +186,269 @@
         }
     }
 }
+
+.tour-dialog {
+    transition: unset;
+    .dc-dialog__dialog {
+        transform: unset;
+        opacity: unset;
+        transition: unset;
+
+        @include mobile {
+            width: 90vw;
+            padding: 1.6rem;
+        }
+        @include tablet {
+            width: 90vw;
+            padding: 1.6rem;
+        }
+    }
+
+    .dc-dialog__content {
+        &__header {
+            text-align: left;
+            margin-bottom: 2.4rem;
+        }
+
+        &__description {
+            text-align: left;
+
+            &__text {
+                margin-bottom: 1.6rem;
+            }
+        }
+    }
+
+    .dc-dialog__footer {
+        @include mobile {
+            display: flex;
+            flex-direction: column;
+            height: unset;
+            align-content: flex-end;
+
+            button {
+                &:first-child {
+                    margin-bottom: unset;
+                    margin-right: 0.8rem;
+                }
+            }
+        }
+    }
+}
+
+.tutorials-wrapper {
+    width: 100%;
+    background: var(--general-main-1);
+    .dc-tabs {
+        &__wrapper {
+            padding: 1.6rem 0.8rem;
+
+            &__group {
+                display: flex;
+                width: 22.5rem;
+                position: relative;
+                justify-content: center;
+                align-items: center;
+                padding: 1.6rem 0;
+                z-index: 1;
+
+                .search-icon {
+                    cursor: pointer;
+                    position: absolute;
+                    left: 1.5rem;
+                    z-index: 1;
+                }
+                .close-icon {
+                    cursor: pointer;
+                    position: absolute;
+                    right: 1.5rem;
+                    z-index: 1;
+                }
+
+                &__search-input {
+                    width: 21rem;
+                    height: 3.2rem;
+                    border-radius: 0.4rem;
+                    outline: none;
+                    background-color: transparent;
+                    font-size: 1.4rem;
```

Comment:
does the search input have a Text component under it? if so can we set the font size there?

Reply:
this is to set the size of the placeholder text this css is used for input type there is no text component used here!